libbeat/docs/release-notes/breaking/breaking-7.16.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[breaking-changes-7.16]]
+
+=== Breaking changes in 7.16
+++++
+<titleabbrev>7.16</titleabbrev>
+++++
+
+See the <<release-notes,release notes>> for a complete list of changes,
+including changes to beta or experimental functionality.
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+==== {journalbeat} is deprecated in 7.16
```

Comment:
Is deprecated enough? It was deleted completely in 7.16. Users cannot download it anymore.

Reply:
good point! I've changed this to say "removed"